lightning/src/util/logger.rs

```diff
@@ -153,8 +153,12 @@ pub trait Logger {
 
 /// Wrapper object to add relevant information to the [`Record`]
 pub struct WithContext<'a, L: Deref> where L::Target: Logger {
+	/// The wrapped [`Logger`] object.
 	pub logger: &'a L,
+	/// The pubkey of peer pertaining to the logging.
 	pub peer_id: Option<PublicKey>,
+	/// The channel id of the channel pertaining to the logging.
+	/// Note: could be a temporary `channel_id` or real `channel_id`
```

Comment:
Looks like these are in the wrong commit.

Reply:
will fix